teuthology/suite/test/test_run_.py

```diff
@@ -360,3 +360,52 @@ def test_newest_success(
         m_find_git_parents.assert_has_calls(
             [call('ceph', 'ceph_sha1', 10)]
         )
+
+    def test_dupa(
+        self
+    ):
+        X = run.descr_to_yamls(
+            "rados/objectstore/{backends/objectstore-bluestore-b supported-random-distro$/{ubuntu_latest}}",
+            "/cephfs/github.com_ceph_build/qa/suites"
+        )
+        assert(X ==
+            [
+               ('rados/objectstore/{backends/objectstore-bluestore-b supported-random-distro$/{ubuntu_latest}}',
+                ['/cephfs/github.com_ceph_build/qa/suites/rados/objectstore/backends/objectstore-bluestore-b.yaml',
+                 '/cephfs/github.com_ceph_build/qa/suites/rados/objectstore/supported-random-distro$/ubuntu_latest.yaml'])
+            ]
+        )
+
+        X = run.descr_to_yamls(
+            "rados/singleton-nomsgr/{all/health-warnings mon_election/connectivity rados supported-random-distro$/{ubuntu_latest}}",
+            "/cephfs/home/akupczyk/src/git.ceph.com_ceph-c_aclamk-testing/qa/suites"
+        )
+        print(str(X))
+        assert(X ==
+            [
+                ("rados/singleton-nomsgr/{all/health-warnings mon_election/connectivity rados supported-random-distro$/{ubuntu_latest}}",
+                 ['/cephfs/home/akupczyk/src/git.ceph.com_ceph-c_aclamk-testing/qa/suites/rados/singleton-nomsgr/all/health-warnings.yaml',
```

Comment:
I think for test cases data it is better to use impersonal path naming, but I am not hundred percent sure about this, don't thank.